hack/make-rules/verify.sh

```diff
@@ -106,10 +107,13 @@ function is-quick {
 function is-explicitly-chosen {
   local name="${1#verify-}"
   name="${name%.*}"
+  index=0
   for e in ${WHAT}; do
     if [[ "${e}" == "${name}" ]]; then
+      TARGET_LIST[${index}]=""
       return
     fi
+    index=`expr $index + 1`
```

Comment:
Thanks @YoubingLi 
added some minor comments:

```
In expr_test.sh line 116:
    index=`expr $index + 1`
          ^-- SC2006: Use $(..) instead of legacy `..`.
           ^-- SC2003: expr is antiquated. Consider rewriting this using $((..)), ${} or [[ ]].
```

Reply:
Sure.

    Following code will be used.
                  index=$(($index + 1))